packages/eslint-plugin-ckeditor5-rules/lib/rules/use-require-for-debug-mode-imports.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const DEFAULT_ALIAS_IMPORT = /import { default as (.*) } from (.*)?;/;
+const NAMED_ALIAS_IMPORT = /import { (.*) as (.*) } from (.*)?;/;
+const NAMED_IMPORT = /import { (.*) } from (.*)?;/;
+const NAMESPACE_ALIAS_IMPORT = /import .* as (.*) from (.*)?;/;
+const DEFAULT_IMPORT = /import (.*) from (.*)?;/;
+const SIDE_EFFECT_IMPORT = /import (.*)?;/;
```

Comment:
Currently, the following debug import is not fixed as we expect, because only the first import is replaced but the second one is not:

```
/**
 * @if CK_DEBUG // import defaultExport from 'module-name';
 * @if CK_DEBUG // import defaultExport2 from 'module-name2';
 */
```

```suggestion
const DEFAULT_ALIAS_IMPORT = /import { default as (.*) } from (.*)?;/g;
const NAMED_ALIAS_IMPORT = /import { (.*) as (.*) } from (.*)?;/g;
const NAMED_IMPORT = /import { (.*) } from (.*)?;/g;
const NAMESPACE_ALIAS_IMPORT = /import .* as (.*) from (.*)?;/g;
const DEFAULT_IMPORT = /import (.*) from (.*)?;/g;
const SIDE_EFFECT_IMPORT = /import (.*)?;/g;
```

Reply:
I do not think this expression is correct at all. See: https://github.com/ckeditor/ckeditor5/blob/a1f8243e0bdfedf3e71d91e64c390c8f3873086f/packages/ckeditor5-engine/src/model/operation/attributeoperation.ts#L207-L211.